File: POOJAVA/src/clasesPOO/Facultad.java
package clasesPOO;

public class Facultad 
{
    String idFacu;
    String NFacu;
    String TelfFacu;
    String IdUni;

    public Facultad(String idFa,String NFa)
    {
        establecerDatos(idFa,NFa);
    }
    public String ObtenerIdFacu()
    {
       return idFacu;
    }
    public String ObtenerFacu()
    {
       return NFacu;
    }
    void  establecerDatos(String idFac,String NFac)
    {
        idFacu=idFac;
        NFacu=NFac;
    }
}

File: POOPHP/app/Model/Facultad.php
<?php
namespace App\Model;
class Facultad
{
    public $idFacu="";
    public $NFacu="";
    public $TelfFacu="";
    public $IdUni="";

   public function __construct(String $idFa,String $NFa)
    {
        $this->establecerDatos($idFa,$NFa);
    }
    public function ObtenerIdFacu()
    {
        return $this->idFacu;
    }
    public function ObtenerFacu()
    {
        return $this->NFacu;
    }
    public function establecerDatos(String $idFac,String $NFac)
    {
        $this->idFacu=$idFac;
        $this->NFacu=$NFac;
    }
}
?>